### ISSUE TITLE
Email verification error

### ISSUE THREAD

**Issue by NicholasBentley** [milesmcc/shynet#75]
I get an `ssl.SSLError: [SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1123)` error when I try to request a verification email.

I think the issue may be that Shynet is trying to connect to the SMTP server using SSL, I didn't notice a config in the .env file that allowed one to specify which method (SSL/TLS) Shynet should use to connect.

After some digging, found the relevant bit here: https://github.com/milesmcc/shynet/blob/fecea17a9d7aa7dffd5101f2968c8aa8c2c71ad9/shynet/shynet/settings.py#L282

I added two more environment variables & their counterparts in settings.py, leaving one commented out sets the correct config. In my build, this produced the expected result; verification email was successfully sent via SMTP using TLS.

`shynet/shynet/shynet/settings.py` (reminds me of inception)
```
# Email

SERVER_EMAIL = os.getenv("SERVER_EMAIL", "Shynet <noreply@shynet.example.com>")
DEFAULT_FROM_EMAIL = SERVER_EMAIL

if DEBUG or os.environ.get("EMAIL_HOST") is None:
    EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
else:
    EMAIL_HOST = os.environ.get("EMAIL_HOST")
    EMAIL_PORT = int(os.environ.get("EMAIL_PORT", 465))
    EMAIL_HOST_USER = os.environ.get("EMAIL_HOST_USER")
    EMAIL_HOST_PASSWORD = os.environ.get("EMAIL_HOST_PASSWORD")
    EMAIL_USE_SSL = os.environ.get("EMAIL_USE_SSL")
    EMAIL_USE_TLS = os.environ.get("EMAIL_USE_TLS")
```

`TEMPLATE.env`
```
# Email settings (optional)
EMAIL_HOST_USER=example
EMAIL_HOST_PASSWORD=example_password
EMAIL_HOST=smtp.example.com
EMAIL_PORT=465
EMAIL_USE_SSL=True
# Comment out EMAIL_USE_SSL & uncomment EMAIL_USE_TLS if your SMTP server uses TLS.
# EMAIL_USE_TLS=True
SERVER_EMAIL=Shynet <noreply@shynet.example.com>
```

Also noticed in the env file the format for SERVER_EMAIL is flopped, fixed that too. Should be `Shynet <noreply@shynet.example.com>` vs `<Shynet> noreply@shynet.example.com`. Django couldn't parse the email correctly in the current format.

If this works happy to open a PR, if there's a better way to fix it I'm all ears.

**Comment by milesmcc** [comment]
Yeah, this is a good fix all around. I took a look at your fork -- would you consider opening a pull request? This is a simple bug fix so I'd happily merge.

**Comment by NicholasBentley** [comment]
@milesmcc no problem. I'll open shortly.

**Comment by milesmcc** [comment]
Fixed in your PR (#76). Thanks!
